client/protocols/guidance.py

```diff
@@ -0,0 +1,178 @@
+from client.protocols.common import models
```

Comment:
Also, some definitions of frequently used terms/acronyms would be helpful:

* TG
* KI
* misU